tests/Console/ConfigurationResolverTest.php

```diff
@@ -1308,3 +1328,11 @@ private function createConfigurationResolver(array $options, Config $config = nu
         );
     }
 }
+
+class MyCustomDiffer implements \PhpCsFixer\Differ\DifferInterface
```

Comment:
can you move this to its own file and use an use-statement for the interface? thanks!

Reply:
Yea sure :)